packages/cli/src/commands/rollback.ts

```diff
@@ -0,0 +1,122 @@
+import chalk from 'chalk';
+import type Client from '../util/client';
+import { ensureLink } from '../util/link/ensure-link';
+import getArgs from '../util/get-args';
+import { getPkgName } from '../util/pkg-name';
+import handleError from '../util/handle-error';
+import logo from '../util/output/logo';
+import ms from 'ms';
+import requestRollback from '../util/rollback/request-rollback';
+import rollbackStatus from '../util/rollback/status';
+import validatePaths from '../util/validate-paths';
+
+const help = () => {
+  console.log(`
+  ${chalk.bold(
+    `${logo} ${getPkgName()} rollback`
+  )} [deploymentId|deploymentName]
+
+  Quickly revert back to a previous deployment.
+
+  ${chalk.dim('Options:')}
+
+    -h, --help                     Output usage information
+    -A ${chalk.bold.underline('FILE')}, --local-config=${chalk.bold.underline(
+    'FILE'
+  )}   Path to the local ${'`vercel.json`'} file
+    -Q ${chalk.bold.underline('DIR')}, --global-config=${chalk.bold.underline(
+    'DIR'
+  )}    Path to the global ${'`.vercel`'} directory
+    -d, --debug                    Debug mode [off]
+    -t ${chalk.bold.underline('TOKEN')}, --token=${chalk.bold.underline(
+    'TOKEN'
+  )}        Login token
+    --timeout=${chalk.bold.underline(
+      'TIME'
+    )}                 Time to wait for rollback completion [3m]
+    -y, --yes                      Skip questions when setting up new project using default scope and settings
+
+  ${chalk.dim('Examples:')}
+
+  ${chalk.gray('–')} Show the status of any current pending rollbacks
+
+    ${chalk.cyan(`$ ${getPkgName()} rollback`)}
+    ${chalk.cyan(`$ ${getPkgName()} rollback status`)}
+    ${chalk.cyan(`$ ${getPkgName()} rollback status --timeout 30s`)}
+
+  ${chalk.gray('–')} Rollback a deployment using id or url
+
+    ${chalk.cyan(`$ ${getPkgName()} rollback <deploymnent id/url>`)}
```

Comment:
I've noticed in that most other places we use square brackets to denote a variable to entered and use `-` in spaces. I'm not sure if we have standard for `or` though. Examples:

https://vercel.com/docs/cli/alias 👉🏾 `vercel alias set [deployment-url] [custom-domain]`
https://vercel.com/docs/cli/list 👉🏾 `vercel list [project-name] [--meta foo=bar]`

So I think for consistency this could be:

Reply:
Angle brackets denotes required and square brackets denotes optional. When `vc rollback` is executed without a deployment id/url, it polls the status and in that context deployment id/url is optional.

However in this particular case, the context is an _example_ of how to perform a rollback in which case the deployment id/url is required and hence angle brackets.

Reply:
Ah cool thank you @cb1kenobi 🙇🏾  It's interesting that the docs say:

```
vercel alias set [deployment-url] [custom-domain]
```

And the CLI for `vc alias --help` says:

```
 set   <deployment> <alias>   Create a new alias
```

I was only looking at the docs and not the other help files. Perhaps this when we get to the idea of syncing our help text with docs this will all get fixed up 🙇🏾

Reply:
I'm of the opinion that the CLI is correct and the docs are not.